src/wallet/wallet.cpp

```diff
@@ -2613,7 +2614,7 @@ SigningResult CWallet::SignMessage(const std::string& message, const PKHash& pkh
     return SigningResult::PRIVATE_KEY_NOT_AVAILABLE;
 }
 
-bool CWallet::FundTransaction(CMutableTransaction& tx, CAmount& nFeeRet, int& nChangePosInOut, bilingual_str& error, bool lockUnspents, const std::set<int>& setSubtractFeeFromOutputs, CCoinControl coinControl)
+bool CWallet::FundTransaction(CMutableTransaction& tx, CAmount& nFeeRet, int& nChangePosInOut, bilingual_str& error, bool includeUnsafe, bool lockUnspents, const std::set<int>& setSubtractFeeFromOutputs, CCoinControl coinControl)
```

Comment:
I suggest adding `includeUnsafe` into `CCoinControl` instead.

Reply:
I completely agree, I mentioned that in the PR description. If there's a concept ACK, `CCoinControl` would be a good place to store that bool. Do you prefer `includeUnsafe` (more consistent with the RPC) or `fSafeOnly` (more consistent with the inner codebase)?

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/21359/commits/0dc8808230160e2da2b2f5aa943f5a3bac7dcb60